server/api/scorepads.js

```diff
@@ -2,7 +2,7 @@ const router = require('express').Router();
 const scorepads = require('../db/scorepads');
 
 router.get('/', (req, res) => {
-  scorepads.get((result) => {
+  scorepads.get(req.query.game, (result) => {
```

Comment:
Ich glaube wir könnten hier einfach nur `req.query` übergeben, dann können wir einfach generisch nach allen Parametern filtern. Wenn man dann `?game` weglässt, werden wie bisher alle `scorepads` zurückgegeben.
```suggestion
  scorepads.get(req.query, (result) => {
```